code/__HELPERS/text.dm

```diff
@@ -456,32 +456,32 @@ proc/checkhtml(var/t)
 
 
 // Pencode
-/proc/pencode_to_html(text, mob/user, obj/item/weapon/pen/P = null, sign = 1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT)
-	text = replacetext(text, "\n",			"<BR>")
-	text = replacetext(text, "\[center\]",	"<center>")
-	text = replacetext(text, "\[/center\]",	"</center>")
-	text = replacetext(text, "\[br\]",		"<BR>")
+/proc/pencode_to_html(text, mob/user, obj/item/weapon/pen/P = null, format = 1, sign = 1, fields = 1, deffont = PEN_FONT, signfont = SIGNFONT, crayonfont = CRAYON_FONT)

```

Comment:
Jesus christ I expected this to be a comment on my bonfire PR for /tg/, scared me a moment. Anyway, I do believe, with the way this is set up, that it defaults to that value unless other calls of `pencode_to_html` override it, but I can double check.